node/src/transition_frontier/candidate/transition_frontier_candidate_actions.rs

```diff
@@ -22,10 +20,11 @@ pub type TransitionFrontierCandidateActionWithMetaRef<'a> =
 #[derive(Serialize, Deserialize, Debug, Clone, ActionEvent)]
 #[action_event(level = debug, fields(debug(hash), debug(error)))]
 pub enum TransitionFrontierCandidateAction {
-    #[action_event(level = info)]
+    P2pBestTipUpdate {
+        best_tip: BlockWithHash<Arc<MinaBlockBlockStableV2>>,
```

Comment:
@binierbtw isn't this the same as ArcBlockWithHash?

Reply:
yeah